src/simpleTrianglePatterns/simpleTrianglePattern.java

```diff
@@ -6,11 +6,11 @@ public static void main(String[] args){
         Scanner userInputScanner = new Scanner(System.in);
         System.out.print("Enter a number:");
         try{
-            int userInputScanner = scan.nextInt();
+            int number = userInputScanner.nextInt();
         } catch (Exception e) {
```

Comment:
How Exceptions works in java? What's the point with the inheritance? How catch works with exceptions? I'm sure this catch case just too broad. What if you got a runtime exception and the error message still say there's a wrong input?

Reply:
bruhhhh